src/trigger_test_instance.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+repo=$1
+
+git config --global user.email "circleci-bot@greenpeace.org"
+git config --global user.name ":robot: CI Bot"
+git config --global push.default simple
+git config --global url."ssh://git@github.com".insteadOf "https://github.com" || true
+mkdir -p ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >~/.ssh/config
+
+instance_repo=planet4-test-$(cat /tmp/workspace/test-instance)
+
+git clone "https://github.com/greenpeace/$instance_repo"
+
+composer_file="$instance_repo/composer-local.json"
+
+zip_url="https://storage.googleapis.com/$(cat /tmp/workspace/zip-path)"
+
+composer_json=$(jq '{name,type,autoload,extra}' </tmp/workspace/composer.json |
+  jq '.version = 7' |
+  jq ".dist = {type: \"zip\", url: \"$zip_url\"}")
+
+package_repo="{type: \"package\", package: $composer_json}"
+
+get_other_repos=".repositories | map(select( .package.name != \"greenpeace/$repo\" ))"
+
+other_repos=$(jq "$get_other_repos" <"$composer_file")
```

Comment:
This will raise an error on current composer-local files that don't have a `.repositories` entry, should we update all test instances ?

Reply:
Good catch, still needed to account for that here, no need to update test instances :+1:

Reply:
Fixed it with ` // []` https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/2474/workflows/a3c12c14-efff-4ea2-9902-84979a2bb288/jobs/12533